tests/test_formatters.py

```
@@ -206,524 +206,526 @@
+    with pytest.raises(JVF):
+        hms_format_from_sec(86411)
+
+    # Errors
+    with pytest.raises(ValueError):
+        format_sec(0, kind='X')
+    with pytest.raises(ValueError):
+        format_sec(0, buffer=np.empty((3,), dtype='int'))
+    with pytest.raises(ValueError):
+        format_sec([[0,0],[0,0]], buffer=np.empty((3,3), dtype='U40'))
+
+def test_format_day_sec():
```

Ruff (0.15.6)

[warning] Too many branches (18 > 12)

(PLR0912)